Simplenote/src/main/java/com/automattic/simplenote/PreferencesFragment.java

```diff
@@ -206,6 +206,53 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
                 CharSequence[] entries = sortPreference.getEntries();
                 sortPreference.setSummary(entries[index]);
 
+                if (!sortPreference.getValue().equals(newValue)) {
+                    switch (index) {
```

Comment:
Suggestion (💡): Consider extracting switch case duplicate code to prevent people from making update mistakes. For example:
```
switch (index) {
    case 0:
        trackSortOrder("modified_newest");
        break;
    case 1:
        trackSortOrder("modified_oldest");
        break;
    case 2:
        trackSortOrder("created_newest");
        break;
    case 3:
        trackSortOrder("created_oldest");
        break;
    case 4:
        trackSortOrder("alphabetical_az");
        break;
    case 5:
        trackSortOrder("alphabetical_za");
        break;
}

private void trackSortOrder(String sortOrderLabel) {
    AnalyticsTracker.track(
            AnalyticsTracker.Stat.SETTINGS_SEARCH_SORT_MODE,
            AnalyticsTracker.CATEGORY_SETTING,
            sortOrderLabel
    );
}
```

Reply:
I added the `trackSortOrder` method in 3fd33f8.